examples/heatTransfer/read/heatRead_adios2.cpp

```diff
@@ -109,7 +111,7 @@ int main(int argc, char *argv[])
     double *T = new double[vT->m_AvailableSteps * readsize[0] * readsize[1]];
```

Comment:
Naaaah, template is fine...in fact, templates are supposed to be used in this kind of situations.  You won't break anything as the adios1 example will be intact. The template type will get resolved automatically. It's less invasive than switching from uint64_t to size_t